### PR TITLE
Revert more neustats stuff back to its old behaviour  

### PR DESCRIPTION
**Pull request by NopoTheGamer**

### CODE REVIEW

File: src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/NEUStatsCommand.kt [hunk 1]
```diff
@@ -54,10 +54,13 @@ import javax.management.ObjectName
 class NEUStatsCommand {
     @SubscribeEvent
     fun onCommands(event: RegisterBrigadierCommandEvent) {
-        event.command("stats", "neustats") {
+        event.command("neustats") {
             thenLiteralExecute("modlist") {
                 clipboardAndSendMessage(
                     DiscordMarkdownBuilder()
+                        .also {
```
Comment (romangraef) [inline src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/NEUStatsCommand.kt:61]: Why tho? Like either we just ask for the full thing, if we need the stats, or we dont

Comment (romangraef) [reply src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/NEUStatsCommand.kt:61]: thats not really a bug in either case

Comment (NopoTheGamer) [reply src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/NEUStatsCommand.kt:61]: 1. it's the old behaviour 2. the more stats the better

Comment (romangraef) [reply src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/NEUStatsCommand.kt:61]: 1. i dont care about the old behaviour.
2. i dont really see the reason for adding this. this is the modlist command, lets just keep the modlist the modlist. removing unrelated stats from that was intentional.

**Pull request closed without merging**